plugins/modules/sda_fabric_virtual_networks_workflow_manager.py

```diff
@@ -971,6 +971,23 @@ def create_payload_for_fabric_vlan(self, vlan, fabric_id_list):
 
         return create_vlan_payload_list
 
+    def sda_fabric_vlan_api_request_limit(self):
+        """
+        Retrieves the limit for SDA API to create/update the sda fabric VLAN tasks..
+        Args:
+            self (object): An instance of a class that provides access to Cisco Catalyst Center.
+        Returns:
+            int: The limit for SDA fabric VLAN api's, which is set to 50 by default.
+        Description:
+            This method returns a predefined limit for the number of request that can be processed or retrieved
+            from SDA fabric VLAN api. Currently, the limit is set to a fixed value of 50.
+        """
+
+        sda_fabric_vlan_limit = 50
+        self.log("API request limit is set to '{0}' for the sda fabric VLAN related task.".format(sda_fabric_vlan_limit), "DEBUG")
```

Comment:
As per our discussion, we will make 50 as default value.. Please get the input from playbook..